code/src/components/Thought.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { formatDistance } from 'date-fns'
+
+const Thought = (props) => {
+
+    const addLikeOnHeartClick = (id) => {
```

Comment:
Look correct what you're writing here Fay! 🥇 You're definitely on the right path. You told me you were not 100% done with this liking function. I think that you can add this line after line 9:
`.then(() => updateLikesVariable(id))`

.. And after this create another function, that updates the likes. Something like this:

`const updateLikesVariable = (id) => {
    const updatedThoughts = thoughts.map(thought => {
      if (thought._id === thoughtId) {
        thought.hearts += 1;
      }
      return thought;
    })
    setThoughts(updatedThoughts) // The state is changed with the updated thoughts, based on new number of likes
  };`